TeamViewer.md

```diff
@@ -0,0 +1,30 @@
+# What is TeamViewer?
+TeamViewer is a software application that allows remote control; desktop sharing and file transfer between computers. This enables you to support customers, work with colleagues remotely, and connect to your own device or support friends and family members when needed.
```

Comment:
You could also make use of screenshots and use lines/boxes to highlight certain areas in the screenshots. 

For lerarning how to add images to markdown, please refer https://guides.github.com/features/mastering-markdown/

Reply:
I will take a look at how to do this and will update the document accordingly.

Reply:
Harshit, I tried to add an image and its working fine on VS code, but it won't work here because the images folder is not merged on gitHub yet. I pushed it on to the master branch but it's still not showing me "compare merge pull request".

Reply:
Actually, this is how it's supposed to work.

Next time, we connect I'll share some tips on making the life of a reviewer easier :)